build.gradle

```diff
@@ -101,6 +100,10 @@ allprojects {
             url "https://maven.research.rackspacecloud.com/content/groups/aggregate/"
         }
     }
+
+    ext {
+        docsHome = "http://www.openrepose.org/versions/$version/index.html".toString()
+    }
```

Comment:
Oh, I see. Was this just done to get the version in scope? Does the version not get inherited from the parent? Could that possible be extracted out of the `allprojects` block?